Net/SMTP.php

```diff
@@ -1475,4 +1529,27 @@ public function identifySender()
     {
         return true;
     }
+
+    /**
+     * Backwards-compatibility method.
+     * array_filter alternative in PHP5.4 for using
+     * key filter because array_filter mode parameter
+     * is only available since PHP5.6.
+     *
+     * @param string $filter The string to filter
+     * @return array Filtered constants array.
+     *
+     * @since 1.10.2
```

Comment:
We don't need `@since` here because it's not part of the public API.

Reply:
ah i see. thanks. i deleted @since. can you have a look again?